Nodejs/Product/Nodejs/Repl/VsNodejsReplSite.cs

```diff
@@ -20,16 +20,26 @@ namespace Microsoft.NodejsTools.Repl {
     class VsNodejsReplSite : INodejsReplSite {
         internal static VsNodejsReplSite Site = new VsNodejsReplSite();
 
-        #region INodejsReplSite Members
+#region INodejsReplSite Members
 
         public CommonProjectNode GetStartupProject() {
-            return NodejsPackage.GetStartupProject(NodejsPackage.Instance);
+            var nodeJsInstance = NodejsPackage.Instance;
+            if (nodeJsInstance == null) {
+                return null;
+            }
+            return NodejsPackage.GetStartupProject(nodeJsInstance);
         }
 
         public bool TryGetStartupFileAndDirectory(out string fileName, out string directory) {
-            return NodejsPackage.TryGetStartupFileAndDirectory(NodejsPackage.Instance, out fileName, out directory);
+            var nodeJsInstance = NodejsPackage.Instance;
```

Comment:
leave a comment here to make it clear in which scenario NodejsPacakge.Instance will be null